asaps/models.py

```diff
@@ -6,12 +6,15 @@
 import csv
 import datetime
 import time
-import copy
-import hashlib
+import logging
+import logging.config
 
 op = operator.attrgetter('name')
 Field = partial(attr.ib, default=None)
 
+logging.config.fileConfig(fname='logging.cfg', disable_existing_loggers=False)
+logger = logging.getLogger('asaps_logger')
```

Comment:
The most common naming scheme for loggers uses the fully qualified module name. In this case here it would be `asaps.models`. The best way to handle this is to just do the following:
```python
logger = logging.getLogger(__name__)
```
The reasons to use this naming scheme are partly to help ensure your logger has a unique name, but also because the logging module uses a namespace hierarchy separated by dots. So you might end up with loggers called `asaps.models`, `asaps.cli`, etc. and they can all be configured once on the parent logger `asaps`.

Reply:
Fixed. And yes, I think that will be a useful conversation about logging, particularly when I add functions that produce reports